### ISSUE TITLE
Data Objects query per Data Object URL doesn't work

### ISSUE THREAD

**Issue by pchainho** [reTHINK-project/dev-registry-domain#56]
As specified [here](https://github.com/reTHINK-project/specs/blob/master/messages/registration-messages.md#query-per-registry-data-object-url)

**Comment by ruimangas** [comment]
We need more information in order to sort this out. I've just tried it (using the REST endpoints as specified [here](https://github.com/reTHINK-project/dev-registry-domain/blob/master/docs/DomainRegistryUserManual.md#rest-api-definition-and-available-endpoints)) and it works.

What kind of messages are being sent? How was this error found? Using directly the Domain Registry or the message node?

**Comment by pchainho** [comment]
Using Msg Node branch dev-interworking with Domain Registry branch develop. 

The message we are sending is:

```
        let message = {
          type: 'read',
          from: _this.registryURL,
          to: 'domain://registry.' + _this._domain + '/',
          body: {
            resource: dataObjectURL
          }
        };
```

**Comment by ruimangas** [comment]
@nunofmn can you this message on the connecter side?

**Comment by nunofmn** [comment]
@pchainho there isn't a branch `dev-interworking` in msg-node. I suppose you are referring to the `dev-address-reusage` branch. 

Could you please provide an example message? So I'm able to debug the problem.

Also, is probably safer to use in `package.json` the most recent connector release `R0.7.0`, instead of using the `develop` branch.

**Comment by rjflp** [comment]
Can we close this one? Is it solved @pchainho ?

**Comment by nunofmn** [comment]
It seems this problem is fixed in current release 0.8.0
If the problem still show up, please reopen.
